src/database/mongo/audio/song_segment.py

```diff
@@ -157,7 +162,11 @@ def get_all_in_range(self, from_count: int, to_count: int):
         """
 
         results = []
-        for r in self._db._db[self._dbcol].find().limit(to_count - from_count).skip(from_count):
+        for r in self._db._db[
+                              self._dbcol
+                              ].find().limit(
+                                             to_count - from_count
+                                             ).skip(from_count):
```

Comment:
I think this looks really weird. Don't know how to do it better though

Reply:
Found a different way that my PEP8 linter eats.